service/grpc/codelingo/codelingo.pb.go

```diff
@@ -1336,6 +1000,32 @@ func (x *codeLingoReviewServer) Recv() (*ReviewRequest, error) {
 	return m, nil
 }
 
+func _CodeLingo_Codemod_Handler(srv interface{}, stream grpc.ServerStream) error {
+	return srv.(CodeLingoServer).Codemod(&codeLingoCodemodServer{stream})
+}
+
+type CodeLingo_CodemodServer interface {
+	Send(*Issue) error
+	Recv() (*ReviewRequest, error)
+	grpc.ServerStream
+}
+
+type codeLingoCodemodServer struct {
+	grpc.ServerStream
+}
+
+func (x *codeLingoCodemodServer) Send(m *Issue) error {
+	return x.ServerStream.SendMsg(m)
+}
+
+func (x *codeLingoCodemodServer) Recv() (*ReviewRequest, error) {
+	m := new(ReviewRequest)
+	if err := x.ServerStream.RecvMsg(m); err != nil {
+		return nil, err
+	}
+	return m, nil
+}
+
 func _CodeLingo_Query_Handler(srv interface{}, stream grpc.ServerStream) error {
```

Comment:
don't use underscores in Go names; func _CodeLingo_Query_Handler should be _CodeLingoQueryHandler